### PR TITLE
Use the new copy in the new editor modal - MAILPOET-6330

### PR DESCRIPTION
**Pull request by pavel-mailpoet**
## Description

Use the new and improved texts 

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-6330](https://mailpoet.atlassian.net/browse/MAILPOET-6330)

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

## Preview

[Preview in WordPress Playground](https://account.mailpoet.com/playground/new/branch:beta-dialogue)

_The latest successful build from `beta-dialogue` will be used. If none is available, the link won't work._

[MAILPOET-6330]: https://mailpoet.atlassian.net/browse/MAILPOET-6330?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/newsletters/editor-select-modal.tsx [hunk 1]
```diff
@@ -47,21 +47,27 @@ export function EditorSelectModal({
...
           src={`${MailPoet.cdnUrl}email-editor/new-editor-modal-header.png`}
-          alt={__('New editor', 'mailpoet')}
+          alt={__('Try the new email editor', 'mailpoet')}
           width="324"
           height="130"
         />
       </div>
+      <p>
+        {__(
+          'Get a sneak peek of an early version of the upcoming email design experience and help shape development.',
```
Comment (triple0t) [inline mailpoet/assets/js/src/newsletters/editor-select-modal.tsx]: Would it make sense to use `help shape its development.` here?

e.g., `Get a sneak peek of an early version of the upcoming email design experience and help shape its development.`

**Review by triple0t — COMMENTED** [review]
Looks good.

I updated the PR description because it was missing the JIRA ticket information.

I dropped a comment. Please check below.

File: mailpoet/assets/js/src/newsletters/editor-select-modal.tsx (hunk 1 above)
Comment (pavel-mailpoet) [reply mailpoet/assets/js/src/newsletters/editor-select-modal.tsx]: Thank you, that makes sense, updated

**Review by triple0t — APPROVED** [review]
LGTM

**Pull request merged**